style-rtl.css

```diff
@@ -811,6 +816,8 @@ button:hover,
 .button:hover,
 .faux-button:focus,
 .faux-button:hover,
+.faux-button .more-link:focus,
```

Comment:
Is there a scenario where this is required? Without this addition, the read more buttons seemed to work perfectly fine.

Reply:
Did you test this line with an RTL locale?

I find that without this line, the `.faux-button .more-link` removes the underline and then it never gets added back.